tests-integration/src/tests/instance_test.rs

```diff
@@ -2199,3 +2199,97 @@ WITH(
         }
     }
 }
+
+#[apply(both_instances_cases)]
+async fn test_copy_parquet_map_to_json(instance: Arc<dyn MockInstance>) {
+    let instance = instance.frontend();
+
+    let output = execute_sql(
+        &instance,
+        r#"CREATE TABLE map_json_test (
+            "id" INT,
+            map_data JSON,
+            ts TIMESTAMP TIME INDEX
+        );"#,
+    )
+    .await
+    .data;
+    assert!(matches!(output, OutputData::AffectedRows(0)));
+
+    let parquet_path = find_testing_resource("/tests/data/parquet/map_to_json.parquet");
+    let output = execute_sql(
+        &instance,
+        &format!(
+            "COPY map_json_test FROM '{}' WITH (FORMAT='parquet');",
+            parquet_path
+        ),
+    )
+    .await
+    .data;
+    assert!(matches!(output, OutputData::AffectedRows(5)));
+
+    let output = execute_sql(
+        &instance,
+        "SELECT \"id\", json_to_string(map_data) FROM map_json_test ORDER BY \"id\";",
```

Comment:
Let's output `map_data` column as well
```suggestion
        &instance,
        "SELECT \"id\", json_to_string(map_data), map_data FROM map_json_test ORDER BY \"id\";",
```

Reply:
Hi @Arshdeep54, it's not hardcoded. For example,
`40000003100000011000000110000001100000011000000110000005616263313268656c6c6f`
is the hex-encoded binary representation of `{"a":"1", "b":"2", "c":"hello"}` in JSONB format.